azure-pipelines.yml

```diff
@@ -38,55 +38,33 @@ variables:
 
 jobs:
 
-- job: vs2017
-  timeoutInMinutes: 120
-  pool:
-    vmImage: VS2017-Win2016
-
-  strategy:
-    matrix:
-        vc2017x86ninja:
-          arch: x86
-          compiler: msvc2017
-          backend: ninja
-          MESON_RSP_THRESHOLD: 0
-        vc2017x64vs:
-          arch: x64
-          compiler: msvc2017
-          backend: vs2017
-        clangclx64ninja:
-          arch: x64
-          compiler: clang-cl
-          backend: ninja
-
-  steps:
-  - task: UsePythonVersion@0
-    inputs:
-      versionSpec: '3.7'
-      addToPath: true
-      architecture: 'x64'
-  - task: PowerShell@2
-    inputs:
-      targetType: 'filePath'
-      filePath: .\ci\run.ps1
-    env:
-      MESON_CI_JOBNAME: azure-$(System.JobName)
-
 - job: vs2019
   timeoutInMinutes: 120
   pool:
     vmImage: windows-2019
 
   strategy:
     matrix:
+        vc2019x86ninja:
+          arch: x86
+          compiler: msvc2019
+          backend: ninja
         vc2019x64ninja:
           arch: x64
           compiler: msvc2019
           backend: ninja
+        vc2019x86vs:
+          arch: x86
+          compiler: msvc2019
+          backend: ninja
```

Comment:
Is this a typo and it's supposed to use the vs2019 backend?

Reply:
Yes, this is a typo